libs/clients/regulations/src/lib/regulations.ts

```diff
@@ -45,6 +58,45 @@ export class RegulationsService extends RESTDataSource {
     this.memoizedResults.clear()
     request.headers.set('Content-Type', 'application/json')
   }
+
+  async createPresignedPost(
+    fileName: string,
+    regId: string,
+    hash?: string,
+  ): Promise<PresignedPostResults | null> {
+    const body = { fileName, hash }
+
+    let response: PresignedPost | null
+    try {
+      response = await this.post<PresignedPost | null>(
+        `file-presigned?scope=${regId}`,
+        JSON.stringify(body),
+        {
+          headers: {
+            'X-ApiKey': this.options.presignedKey ?? '',
+          },
+        },
+      )
```

Comment:
Not sure what this is doing. If it´s sensitive, has it been security reviewed?

Reply:
It is not sensitive. 🤝 
This is actually a temporary solution until we move this upload over to the Island.is infrastructure.